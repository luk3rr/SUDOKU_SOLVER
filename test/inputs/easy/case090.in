000020403 006000002 003040097 200008009 000006050 009400000 700504000 000900740 300001080 
