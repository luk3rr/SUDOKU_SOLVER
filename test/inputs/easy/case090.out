Solving the following grid:
0 0 0 | 0 2 0 | 4 0 3 
0 0 6 | 0 0 0 | 0 0 2 
0 0 3 | 0 4 0 | 0 9 7 
------+-------+------
2 0 0 | 0 0 8 | 0 0 9 
0 0 0 | 0 0 6 | 0 5 0 
0 0 9 | 4 0 0 | 0 0 0 
------+-------+------
7 0 0 | 5 0 4 | 0 0 0 
0 0 0 | 9 0 0 | 7 4 0 
3 0 0 | 0 0 1 | 0 8 0 


Solution found :')

9 5 8 | 1 2 7 | 4 6 3 
4 7 6 | 8 3 9 | 5 1 2 
1 2 3 | 6 4 5 | 8 9 7 
------+-------+------
2 4 5 | 7 1 8 | 6 3 9 
8 1 7 | 3 9 6 | 2 5 4 
6 3 9 | 4 5 2 | 1 7 8 
------+-------+------
7 9 1 | 5 8 4 | 3 2 6 
5 8 2 | 9 6 3 | 7 4 1 
3 6 4 | 2 7 1 | 9 8 5 

Algorithm: BFS
Total time: 147 ms
Total expanded states: 19117
